frontend/src/metabase/forms/components/FormGroupsWidget/FormGroupWidget.tsx

```diff
@@ -0,0 +1,107 @@
+import type { HTMLAttributes } from "react";
+import { t } from "ttag";
+import _ from "underscore";
+import { useField } from "formik";
+import MembershipSelect from "metabase/admin/people/components/MembershipSelect";
+import { useGroupListQuery } from "metabase/common/hooks";
+import FormField from "metabase/core/components/FormField";
+
+import { isDefaultGroup, isAdminGroup } from "metabase/lib/groups";
+
+import type { GroupId, Member } from "metabase-types/api";
+
+interface FormGroupsWidgetProps extends HTMLAttributes<HTMLDivElement> {
+  name: string;
+}
+
+export const FormGroupsWidget = ({
```

Comment:
Because this was only used in the one place, I migrated it to grab it's value via useField, rather than have it passed in via props, as well as use hooks to fetch data rather than HoCs